All deskop project team pages

LudvigStridsberg opened the pull request:
Created all desktop versions of the different about the team-pages.

### Development page
![Screenshot_2021-04-14 Snatch - The development team](https://user-images.githubusercontent.com/77850704/114701098-a6ee0e00-9d22-11eb-8ad4-1b0eed98232d.png)

### UX page
![Screenshot_2021-04-14 Snatch - The UX team](https://user-images.githubusercontent.com/77850704/114701145-b4a39380-9d22-11eb-963e-dfa4716a6320.png)

### Marketing page
![Screenshot_2021-04-14 Snatch - The marketing team](https://user-images.githubusercontent.com/77850704/114701175-be2cfb80-9d22-11eb-8371-a76ae6969652.png)

LudvigStridsberg committed:
Desktop team pages added

Created all desktop versions of the different team pages. Put the needed images in the images-folder. Linked the pages from the About the team-page. Also added the required code into app.js.

LudvigStridsberg committed:
Added responsive desktop-css

Added some css so that the pages are more responsive while still in desktop-mode.

VilmaNilsson commented on public/css/about-teams.css:
```diff
@@ -0,0 +1,97 @@
+
+ h1 {
+    font-family: var(--rajdhaniFont);
+    font-weight: normal;
```
Everything looks amazing! Just one tiny thing, change the text here to numbers instead. Here it should be 500. :D

LudvigStridsberg replied:
Requested changes has been implemented.

VilmaNilsson reviewed (changes requested):
All looks amazing! I would like on _tiny tiny_ change, make the name text (ex "Ludvig" or "Vilma") a bit thicker.

LudvigStridsberg committed:
Font-weight and margin change

Changed the weight of the headers to 500 as requested by Vilma.

Also made a small adjustment to the margin below the logo.

Co-Authored-By: Vilse <77903212+VilmaNilsson@users.noreply.github.com>
